### DISCUSSION TITLE
Enums support?

### DISCUSSION THREAD

**Discussion by hamzaelmaghari** [spatie/laravel-settings#159] [Q&A] [answered]
Is the package supports enums or not? If yes, how?

**Comment by rubenvanassche** [comment, marked as answer]
You just use an enum within your settings class, take a look at the tests if you need more context:

https://github.com/spatie/laravel-settings/blob/ee1b1a01e02a9305488620d6c17312d448ae628d/tests/SettingsTest.php#L640-L668
